src/WatchModel.php

```diff
@@ -30,10 +34,44 @@ public function getAllWatches()
                 $watch['brand'], 
                 $watch['model_name'],
                 $watch['dial_colour'],
-                $watch['movement']
+                $watch['movement'],
+                $watch['deleted']
             );
         }
-        
         return $watchObjs;
     }
+
+    public function saveNewWatchType($brand ,$dial_colour, $model_name, $movement)
```

Comment:
You should add type hints to the paremeters here, and the return type should be bool (because you return execute, that returns bool)